src/index.pug

```diff
@@ -0,0 +1,18 @@
+doctype html
+html
+    head
+        meta(name="viewport" content="width=device-width")
+        meta(name="description" content="This is the version 2 of My-cv for junior and we gonna use pug as templete engine for hrml")
+        title Welcome to My CV Version 2 Junior
+        //- link(rel="stylesheet", href="./style.scss")
+        if config.entry.css.embed
```

Comment:
if and the line after it are at the same level. there is no purpose for this.
indent the next one so it gets affected by the if condition